specs/src/specs/header-sync.md

```diff
@@ -28,27 +28,27 @@ The sequencer node, upon successfully creating the block, publishes the signed b
 
 ## Assumptions
```

Comment:
[non-blocking]
Why is this spec here? Shouldn't it be in the `block` directory as that is where the header sync code is?

Reply:
Follow up issue here: https://github.com/rollkit/rollkit/issues/1249